components/file_upload/file_upload.test.tsx

```diff
@@ -208,18 +209,19 @@ describe('components/FileUpload', () => {
         event.preventDefault = jest.fn();
         const getAsFile = jest.fn().mockReturnValue(new File(['test'], 'test'));
         const file = {getAsFile, kind: 'file', name: 'test'};
-        event.clipboardData = {items: [file], types: ['image/png']};
+        (event as any).clipboardData = {items: [file], types: ['image/png']};
```

Comment:
I couldnt make this work, so I had to use `any` 😕 Please let me know if you have any suggestion 💡

Reply:
That's probably fine. We're faking a lot of objects here, so I'm not surprised we end up needing `any` for some of them